client/js/libs/jquery/stickyscroll.js

```diff
@@ -16,7 +24,8 @@
 					self.scrollBottom();
 				}
 			})
-			.scrollBottom();
+			.scrollBottom()
+			.data("window-sticky-fn", windowResizeHandler);
```

Comment:
Don't really think we need the data attribute, can you just use a variable in the anonymous scope.